### PR TITLE
netcoreapp2.0

### PR DESCRIPTION
**Pull request by anurse**
Removing net46 targets.

### CODE REVIEW

File: src/Microsoft.VisualStudio.Web.CodeGeneration.Contracts/Microsoft.VisualStudio.Web.CodeGeneration.Contracts.csproj [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <Import Project="..\..\build\common.props" />
   <PropertyGroup>
-    <TargetFramework>netstandard1.3</TargetFramework>
+    <TargetFramework>netcoreapp2.0</TargetFramework>
```
Comment (prafullbhosale) [inline src/Microsoft.VisualStudio.Web.CodeGeneration.Contracts/Microsoft.VisualStudio.Web.CodeGeneration.Contracts.csproj]: Why does this have to move to `netcoreapp2.0`
Shouldn't the libraries remain at `netstandard`?

File: src/Microsoft.VisualStudio.Web.CodeGeneration.Core/Microsoft.VisualStudio.Web.CodeGeneration.Core.csproj [hunk 2]
```diff
@@ -2,11 +2,11 @@
   <Import Project="..\..\build\common.props" />
   <PropertyGroup>
     <Description>Contains the core infrastructure used by ASP.NET Core Code Generators.</Description>
-    <TargetFrameworks>net46;netstandard1.6</TargetFrameworks>
+    <TargetFramework>netcoreapp2.0</TargetFramework>
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
     <NoWarn>$(NoWarn);CS1591</NoWarn>
     <PackageTags>aspnetcore;codegenerator;scaffolding;visualstudioweb</PackageTags>
-    <PackageTargetFallback Condition=" '$(TargetFramework)' == 'netstandard1.6' ">$(PackageTargetFallback);portable-net45+win8</PackageTargetFallback>
+    <PackageTargetFallback>$(PackageTargetFallback);portable-net45+win8</PackageTargetFallback>
```
Comment (muratg) [inline src/Microsoft.VisualStudio.Web.CodeGeneration.Core/Microsoft.VisualStudio.Web.CodeGeneration.Core.csproj]: Do we need this?  cc @prafullbhosale

**Comment by anurse** [comment]
Closing and discussing more offline

File: src/Microsoft.VisualStudio.Web.CodeGeneration.Core/Microsoft.VisualStudio.Web.CodeGeneration.Core.csproj (hunk 2 above)
Comment (prafullbhosale) [inline src/Microsoft.VisualStudio.Web.CodeGeneration.Core/Microsoft.VisualStudio.Web.CodeGeneration.Core.csproj]: Yes, till https://github.com/dotnet/roslyn/issues/16040 is fixed and scaffolding is updated to use the fixed Roslyn packages.

**Pull request closed without merging**